merge_values.go

```diff
@@ -135,6 +135,10 @@ func processInterfaceMap(in map[interface{}]interface{}) map[string]interface{}
 }
 
 func processValue(v interface{}) interface{} {
+	if v == nil {
```

Comment:
What about linking to the upstream docs here?

Reply:
Done